plugins/orchestrator-backend/src/service/OrchestratorService.ts

```diff
@@ -182,6 +182,22 @@ export class OrchestratorService {
       : undefined;
   }
 
+  public async retriggerWorkflow(args: {
+    definitionId: string;
+    instanceId: string;
+    serviceUrl: string;
+    cacheHandler?: CacheHandler;
+  }): Promise<boolean | undefined> {
+    const { definitionId, cacheHandler } = args;
+    const isWorkflowAvailable = this.workflowCacheService.isAvailable(
+      definitionId,
+      cacheHandler,
+    );
+    return isWorkflowAvailable
+      ? await this.sonataFlowService.retriggerInstance(args)
+      : undefined;
```

Comment:
In the past there was an invocation of `orchestratorService.updateInstanceInputData`. I know that there were issue, but can you explain why we don't needed anymore? To be sure we're doing the right thing.

Reply:
This is definitely the right thing
Also discussed it with @masayag so he can confirm
updating the instance with input data is a bug
Consider the following scenario: The user requested to scale up the cluster according to certain specs. He got that approved. Then the scaling failed. Now he can run retrigger, update the scale to much bigger specs, invoke retrigger and the workflow will skip the Jira ticket and go directly to the scale up!

Reply:
To generalize what @batzionb said, editing input parameters during retrigger may enable the re-execution of a workflow with unapproved parameters.

Reply:
so this is resolved?